spatialdata/_core/transformations.py

```diff
@@ -197,10 +197,40 @@ def _get_n_spatial_dims(self, axes: tuple[str, ...]) -> int:
             raise ValueError(f"Invalid axes: {axes}")
         return valid_axes[axes]
 
-    def transform(self, element: SpatialElement) -> SpatialElement:
+    def transform(self, element: SpatialElement, maintain_positioning: bool = False) -> SpatialElement:
+        """
+        Transform a spatial element using this transformation and returns the transformed element.
+
+        Parameters
+        ----------
+        element
+            Spatial element to transform.
+        maintain_positioning
+            If True, each transformation of the transformed element will be prepended with the inverse of the current
+            transformation. In this way the data is transformed but the global positioning is maintained. An use case
+            changing the orientation/scale/etc of the data but keeping the alignment of the data within each coordinate
+            system. Please see notes for more details of how this parameter interact with xarray.DataArray for raster
+            data.
+
+        Returns
+        -------
+        SpatialElement
+            Transformed spatial element.
```

Comment:
this will mace read the docs fail I think, it should be ``SpatialElement``: the transformed spatial element.

Reply:
changed